sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/HiveSqlParser.scala

```diff
@@ -140,7 +140,7 @@ class HiveSqlAstBuilder(conf: SQLConf) extends SparkSqlAstBuilder(conf) {
       format(inRowFormat, "hive.script.recordreader")
 
     val (outFormat, outSerdeClass, outSerdeProps, writer) =
-      format(outRowFormat, "hive.script.recordwriter")
+      format(inRowFormat, "hive.script.recordwriter")
```

Comment:
cc @hvanhovell this is pretty weird -- i think having this be outRowFormat actually breaks some tests.